lighthouse-core/runner.js

```diff
@@ -110,6 +110,8 @@ class Runner {
       // Entering: conclusion of the lighthouse result object
       // @ts-ignore - Needs json require() support
       const lighthouseVersion = /** @type {string} */ (require('../package.json').version);
+      // @ts-ignore - ENVIRONMENT varialble is not declared
```

Comment:
@brendankenny not sure how to best handle that. Should we have a separate ts file with `interface Global`?